.github/workflows/sample-run.yml

```diff
@@ -21,31 +21,19 @@ jobs:
         run: |
           go build -v ./...
 
-      # - name: Setup Java
-      #   uses: actions/setup-java@v3
-      #   with:
-      #     java-version: "11"
-      #     distribution: "temurin"
+  build-arm:
```

Comment:
also you dont have to create everything again just another section 
```

- name: Build Keploy ARM
   run : ........
```

Reply:
Done